docs/Deploy/deploy-on-aws-with-terraform.md

```diff
@@ -295,7 +295,7 @@ During this process, terraform will create a `hosted zone` with an `NS` record f
 
 Let’s say we own `queryops.com` and have an ACM certificate issued to it. We want to host Fleet at `fleet.queryops.com` so in this case, we’ll need to hand nameserver authority over to `fleet.queryops.com` before ACM will verify via DNS and issue the certificate. To make this work, we need to create an `NS` record on `queryops.com` and copy the `NS` records that were created by terraform for the `fleet.queryops.com` hosted zone.
 
-![Route 53 QueryOps Hosted Zone](../website/assets/images/articles/deploying-fleet-on-aws-with-terraform-1-622x250@2x.png)
+![Route 53 QueryOps Hosted Zone](https://fleetdm.com/images/docs/deploy-on-aws-1-622x250@2x.png)
```

Comment:
Noticed the actual file name below has an extra dash (`deploy-on-aws--1-622x250@2x.png`) so either this reference or the file will need to change